### PR TITLE
Navbar update

### PR DESCRIPTION
**Pull request by imogenmisso**

### CODE REVIEW

**Comment by runticle** [comment]
uh oh

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -10,6 +10,10 @@
...
  *
- *= require_tree .
- *= require_self
  */
+
+ @import 'bootstrap';
+
+body {
+  background-color: #e6e9ef
+}
```
Comment (runticle) [inline app/assets/stylesheets/application.scss]: great colour choice

File: app/assets/stylesheets/posts.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Place all the styles related to the Comments controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+div[name="post"] {
```
Comment (runticle) [inline app/assets/stylesheets/posts.scss]: inventive thinking! ✨

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,65 +1,58 @@
...
+        <h4><%= post.user.name.capitalize %></h4>
+        <p id="time"><%= post.updated_at.strftime("%e %b %Y") %></p>
+        <span id="post_content"><%= simple_format(post.message) %></span>
+      </div>
       <% if post.user == current_user %>
         <%= link_to '❌', post_path(post), method: :delete, id: "delete-#{post.id}" %>
-        <% if post.editable? %>
-          <%= link_to '✏️', edit_post_path(post.id) %>
-        <% end %>
+        <% if post.editable? %><%= link_to '✏️', edit_post_path(post.id) %><% end %>
```
Comment (runticle) [inline app/views/posts/index.html.erb]: short and sweet

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -1,65 +1,58 @@
...
-            <%= link_to '✏️', edit_post_comment_path(post.id, comment.id) %>
+            <span class="comment_buttons"><%= link_to '✏️', edit_post_comment_path(post.id, comment.id) %></span>
           <% end %>
         <% end %>
-
-
         <span class="pull-right">
           <%= link_to like_post_comment_path(post, comment), method: 'post', remote: true do %> 👍
-          <span id="Comment_<%= comment.id %>_likes"><%= comment.likes.count %>likes</span>
+            <span id="Comment_<%= comment.id %>_likes"><%= comment.likes.count %>likes</span>
```
Comment (runticle) [inline app/views/posts/index.html.erb]: is there a difference between .count and .size?

File: config/initializers/assets.rb [hunk 5]
```diff
@@ -14,3 +14,4 @@
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
 # Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w( style.css )
```
Comment (runticle) [inline config/initializers/assets.rb]: what is this ?

File: spec/features/posts_reverse_chrono_spec.rb [hunk 6]
```diff
@@ -19,7 +19,7 @@
     click_button 'Submit'
     time_2 = Time.now
     Timecop.return
-
-    expect(page.find('li:nth-child(1)')).to have_content("My second post!")
+    # save_and_open_page
```
Comment (runticle) [inline spec/features/posts_reverse_chrono_spec.rb]: remove this please.

File: spec/features/users_can_register_spec.rb [hunk 7]
```diff
@@ -6,7 +6,7 @@
 RSpec.feature 'Registration', type: :feature do
   scenario 'Can successfully register as a user' do
     signup
-    expect(page).to have_content('Welcome to Acebook Alfie')
+    expect(page).to have_content("Welcome to Acebook, Alfie!")
```
Comment (runticle) [inline spec/features/users_can_register_spec.rb]: thanks acebook!

File: Gemfile.lock [hunk 8]
```diff
@@ -262,7 +272,8 @@ DEPENDENCIES
   factory_bot_rails (~> 4.9)
   faker (~> 1.9)
   jbuilder (~> 2.5)
-  jquery-rails (~> 4.3)
+  jquery-rails
+  launchy (~> 2.4)
```
Comment (imogenmisso) [inline Gemfile.lock]: appears twice

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (imogenmisso) [reply app/assets/stylesheets/application.scss]: my favourite shade of white/blue/grey!!!!!!

File: app/views/posts/index.html.erb [hunk 9]
```diff
@@ -1,65 +1,58 @@
...
 <% end %>
 
-<ul>
+<div class="container" id="posts">
   <% @posts.each do |post| %>
-    <li>
-      <%= simple_format(post.message) %>
-      <p><%= post.user.name %></p>
-      <p><%= post.updated_at.strftime("%eth %b %Y %T") %></p>
+    <div class="border" name="post" id="post_<%= post.id %>">
```
Comment (imogenmisso) [inline app/views/posts/index.html.erb]: 🌟

File: app/views/posts/index.html.erb (hunk 4 above)
Comment (toddpla) [reply app/views/posts/index.html.erb]: should be size

https://stackoverflow.com/questions/6083219/activerecord-size-vs-count

File: app/views/posts/index.html.erb [hunk 10]
```diff
@@ -1,65 +1,58 @@
...
-      <span id="Post_<%= post.id %>_likes"><%= post.likes.count %>likes</span>
-    	<% end %>
-    </span>
-
-
-    <%= link_to new_post_comment_path(post.id) do %>
-      Comment
-    <% end %>
-    <ul>
+      <%= link_to '💬', new_post_comment_path(post.id) %>
```
Comment (imogenmisso) [inline app/views/posts/index.html.erb]: cool comment emoji

File: config/initializers/assets.rb (hunk 5 above)
Comment (imogenmisso) [reply config/initializers/assets.rb]: 🤷🏽‍♀️

Comment (toddpla) [reply config/initializers/assets.rb]: @imogenmisso

**Pull request merged**